src/Symfony/Component/Console/composer.json

```diff
@@ -26,6 +26,7 @@
         "symfony/dependency-injection": "~3.4|~4.0",
         "symfony/lock": "~3.4|~4.0",
         "symfony/process": "~3.4|~4.0",
+        "symfony/var-dumper": "~3.4|~4.0",
```

Comment:
Should we update `suggest` section as well?

Reply:
not sure it's worth it, most likely the component is already available in projects for `dump()`. 

If not, i doubt, the suggest would help the discoverability for this tiny feature/helper :)